<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeacherMainPage</class>
 <widget class="QWidget" name="TeacherMainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>176</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>AcadEref</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Audio input</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_audiodevice">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Default</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="b_togglestate">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>40</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Class begin</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_sharescreen">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>40</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Share screen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_pushprob">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>40</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Push problem</string>
   </property>
  </widget>
  <widget class="QPushButton" name="b_randcall">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>120</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Random roll call</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_called">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="cb_vidwindow">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>221</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Desktop</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Shared screen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
